src/msw/toolbar.cpp

```diff
@@ -1694,8 +1694,50 @@ bool wxToolBar::MSWOnNotify(int WXUNUSED(idCtrl),
                 nmtbcd->clrTextHighlight = wxColourToRGB(GetForegroundColour());
                 nmtbcd->clrHighlightHotTrack = wxSysColourToRGB(wxSYS_COLOUR_HOTLIGHT);
 
-                *result = CDRF_DODEFAULT | TBCDRF_USECDCOLORS | TBCDRF_HILITEHOTTRACK;
+                *result = CDRF_DODEFAULT | TBCDRF_USECDCOLORS | TBCDRF_HILITEHOTTRACK | CDRF_NOTIFYPOSTPAINT;
                 return true;
+
+            case CDDS_ITEMPOSTPAINT: {
+                // custom draw the drop-down arrow here, as it is always black
+                TBBUTTONINFO bi = { 0 };
+                bi.cbSize = sizeof(TBBUTTONINFO);
```

Comment:
For consistency please use
```suggestion
                WinStruct<TBBUTTONINFO> bi;
```